src/Components/CreateImageWizard/utilities/PasswordValidatedInput.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
```

Comment:
There's a following warning in the test output:
```
Text input: Text input requires either an id or aria-label to be specified
```

seems like there's aria-label correctly specified in the PasswordValidatedInput, so this might be because of a different field, but we should take a look at it.

Reply:
Checked again - it's caused by removing the `id` from `<TextInput>`

Reply:
the aria-label also will solve it :)